src/chainstate/stacks/boot/cost-voting.clar

```diff
@@ -0,0 +1,208 @@
+;; The .cost-voting contract
+
+;; error codes
+(define-constant ERR_PROPOSE 1)
+
+;; cost vote token
+(define-fungible-token cost-vote-token)
+
+;; proposal counters
+(define-data-var proposal-count uint u0)
+(define-data-var confirmed-proposal-count uint u0)
+
+;; max confirmed cost-funciton ID
+(define-data-var cost-function-max-id uint u1000)
```

Comment:
Is `cost-function-max-id` used for anything? If not, can you remove the data-var definition?

Reply:
I don't think it's currently being used. I think I may have pasted it in from your branch. Is there a reason we might want to have a max-id?

Reply:
I don't think it's necessary.